src/Cache.php

```diff
@@ -108,16 +112,17 @@ public function has($key): bool
      * Some caches, such as memcached or apcu, allow retrieving multiple cached values at the same time,
      * which may improve the performance. In case a cache does not support this feature natively,
      * this method will try to simulate it.
-     * @param string[] $keys list of string keys identifying the cached values
+     * @param iterable $keys list of string keys identifying the cached values
```

Comment:
Why? `string[]` is more precise - you can't pass list of objects.

Reply:
Interface allows iterables, they are not only arrays.